.github/PROCESS.md

```diff
@@ -53,12 +53,16 @@ This guide tries to be short and does not mention things that are completely aut
 2. Pull request **title** should be accurate and descriptive as it is used in the generated changelog.
    It should not mention the issue number but describe the fixed issue instead.
 3. Most of the time pull requests are related to some issues.
-   In such cases pull request **description** should mention the issue number like `#123`.
+   In such cases pull request **description** should mention the issue number like `Closes #{issue_number}.`.
    Do not use URLs like `https://github.com/org/repo/issue/123` even if they are rendered the same on GitHub.
    If you propose a tiny fix, there is no needed to create a new issue.
 4. We provide a pull requests template that includes suggestions and readiness checklist.
    Please use it.
 5. It is fine to send several sequential pull requests for one issue to make them easier to review.
+   In that case, please still use the `Closes #{issue_number}.` phrase
```

Comment:
```suggestion
   In that case, please use the `Closes #{issue_number}.` phrase
```

Reply:
Why?

Reply:
Made the suggestion based on the sentence tone, could sound better without the use of `still` in this context. But feel free to ignore it. Otherwise, everything else is good.

Reply:
My point was to highlight that even if that PR does not close the issue, we still should say "closes", which sounds counter-intiutive

Reply:
Okay. I understand better now. Will something like this be a bit clearer ....` In that case, even if a particular PR does not close the issue, please use the ...`?

Reply:
I'm merging it as-is now to avoid another full round of reviews, but I will rewrite that sentence in the next pass.